compendium/ServiceComponent/include/cppmicroservices/servicecomponent/ComponentContext.hpp

```diff
@@ -221,6 +241,23 @@ namespace cppmicroservices
                  */
                 virtual std::shared_ptr<void> LocateService(std::string const& name, std::string const& type) const = 0;
 
+                /**
+                 * Returns the service object from the cache matching the passed in
```

Comment:
the cache is an implementation detail that I don't think should be mentioned in the public facing documentation. The user shouldn't care from where the service object is retrieved.